From 5c60c469bb397e1237d0216995ca6c0e0d122e97 Mon Sep 17 00:00:00 2001
From: Chew, Chiau Ee <chiau.ee.chew@intel.com>
Date: Fri, 27 Dec 2013 23:22:20 +0800
Subject: [PATCH 13/17] x86/Kconfig: add PCI dependency for
 CONFIG_X86_INTEL_LPSS

Allow CONFIG_X86_INTEL_LPSS to be set when ACPI
or PCI is set.

Signed-off-by: Chew, Chiau Ee <chiau.ee.chew@intel.com>
---
 arch/x86/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index 14c13c0..4067fc4 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -459,7 +459,7 @@ endif

 config X86_INTEL_LPSS
 	bool "Intel Low Power Subsystem Support"
-	depends on ACPI
+	depends on ACPI || PCI
 	select COMMON_CLK
 	---help---
 	  Select to build support for Intel Low Power Subsystem such as
--
1.7.4.4

